babelize/content.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8; mode:python -*-
+
+import logging
+import os
+
+logger = logging.getLogger(__name__)
+
+
+class ContentFile:
+
+    def __init__(self, path):
+        self.__path = path
+
+    @property
+    def path(self):
+        return self.__path
+
+    @property
+    def translations(self):
+        # TODO - this might need caching as it could be very expensive
+        # for large amount of files.
+        retval = {}
+        for c in self.path.parent.iterdir():
+            if c.is_dir():
+                continue
+            if c.match(f'*{self.path.stem}.??.md'):
+                _, lang, _ = c.name.split('.')
+                retval[lang] = c
+        return retval
+
+    def make_symlink(self, lang):
+        f = self.translations.get(lang)
+        if f is None:
+            target = self.path.with_name(self.path.stem + f'.{lang}.md')
+            os.symlink(self.path.name, target)
+            return target
+        return None
+
+    def __str__(self):
+        return str(self.__path)
+
+
+def make_symlinks(cfg):
```

Comment:
I think this should take a language as an argument so making links is (or at least can be) explicit. I might not want to make links for a new language, or deliberately leave one out.